### PR TITLE
Chat Window revamp

### PR DESCRIPTION
**Pull request by FredyJabe**
Description:
The current chat window seems small and somewhat clunky, so I spent some time working on it to make it more appealing and functional.

Summary:
- New visuals, with the currently connected players list
- Tag functionality to ping others
- Bug fixes (Messages being cut at the end, issues with the scrollbar)
- Some warning messages fixes (because 430 warnings is a lot, I'll get that to 0 eventually)
- Added text formatting for users, bold, italic and colored text using simple tags

The new visual: 
image
image
image

### CODE REVIEW

**Comment by Byte-Nova** [comment]
Looks good from the outside, I'll take some time to review it so feel free to modify it if you find anything to add/fix

File: Source/Client/Managers/ChatManager.cs [hunk 1]
```diff
@@ -155,6 +154,61 @@ public static void ChatClock()
...
+                
+                if (verification != "" && !verifying)
+                {
+                    switch(verification)
+                    {
+                        case "[/]":
+                            if (codeType.Count > 0) parsedMessage = parsedMessage.ReplaceFirst(verification, $"</{codeType.Pop()}>");
+                            verification = "";
+                            break;
+                        case "[b]":
```
Comment (nightred) [inline Source/Client/Managers/ChatManager.cs:188]: This is nice formatting style but I would suggest changing to markdown as it is simpler for the user and does not use bracket notation.

Comment (FredyJabe) [reply Source/Client/Managers/ChatManager.cs:188]: I'll do some tests later today, what about colors in markdown?
I used the bracket system to keep the same style for everything.

Comment (nightred) [reply Source/Client/Managers/ChatManager.cs:188]: you are right markdown does not have color, I forgot this fact.
My choice would be brackets for colors and markdown for formatting, but I am just a user.

Comment (FredyJabe) [reply Source/Client/Managers/ChatManager.cs:188]: There, added a custom markdown for color codes: starts with `&rrggbb` and ends with `&&`

**Pull request merged**